src/codeu/chat/server/View.java

```diff
@@ -46,15 +46,15 @@ public View(Model model) {
 
   @Override
   public Collection<User> getUsers(Collection<Uuid> ids) {
-    return model.userById(intersect(ids, false), "ASC");
+    return model.userById(ids, false, true);
```

Comment:
I don't know if Andre is planing on implementing any method that requires a DESC ordering, so I'll leave it as it is right now. If he doesn't need it later, I'll just remove it when doing some cleanup.